views/js/controller/Tools/assessmentActivity.js

```diff
@@ -21,156 +21,153 @@
-                    transform: function(value) {return value.toString();}
-                },
-                {
-                    id: 'Canceled',
-                    label: __('Canceled'),
-                    sortable : true,
-                    transform: function(value) {return value.toString();}
-                },
-                {
-                    id: 'Finished',
-                    label: __('Finished'),
-                    sortable : true,
-                    transform: function(value) {return value.toString();}
-                },
-            ];
-
-            $deliveryList.datatable({
-                url: url.route('deliveriesActivityData', 'Tools', 'taoProctoring'),
-                filter: false,
-                model: deliveryListModel,
-                paginationStrategyTop : 'none',
-                paginationStrategyBottom : 'none',
-                selectable : true,
-                sortorder : 'asc',
-                sortby : 'label'
-            }, deliveryListModel);
-            $deliveryList.datatable('refresh');
-
-
-            $('.js-pause').on('click', function() {
-                var config;
-
-                config = {
-                    renderTo : $container,
-                    actionName : msg,
-                    reason : true,
-                    allowedResources: [],
-                    reasonRequired: true,
-                    categoriesSelector: cascadingComboBox(categories['pause'])
-                };
+        // Active Proctors
+        $('.active-proctors', $userActivityContainer).text(data.active_proctors);
+    }
 
-                bulkActionPopup(config).on('ok', function(reason){
-                    doPause(reason);
-                });
-            });
+    function updateAssessmentActivity(data) {
+        // Total Current Assessments
+        $('.total-current-assessments', $assessmentActivityContainer).text(data.total_current_assessments);
 
-            $('.js-activity-chart-interval').on('change', function () {
-                var interval = $(this).val();
-                activityGraph.refresh({
-                    graphConfig : {
-                        data : {
-                            url: url.route('completedAssessmentsData', 'Tools', 'taoProctoring', {'interval' : interval})
-                        },
-                        axis : {
-                            x : {
-                                tick : {
-                                    format: interval === 'day' ? '%H:%M' : '%m-%d'
-                                },
-                                label : {
-                                    text:  interval === 'day' ? __('Hours') : __('Days')
-                                }
-                            }
-                        }
-                    }
-                });
-            });
+        // In Progress
+        $('.in-progress-assessments', $assessmentActivityContainer).text(data.in_progress_assessments);
```

Comment:
It is indeed preferable to have all text update in updateAssessmentActivity() done via a handlebars template, in the assessmentActivity component